### DISCUSSION TITLE
Does Gowebly bundle DaisyUI or get from CDN?

### DISCUSSION THREAD

**Discussion by zakkwylde** [gowebly/gowebly#66] [Q&A] [answered]
Does Gowebly bundle DaisyUI or get from CDN? would like to understand how this works. Does everything get packaged into the executable?

**Comment by koddr** [comment, marked as answer]
Hi, @zakkwylde 

First of all, please read this article in the official docs: https://gowebly.org/getting-started/how-does-it-work

All frontend libraries that are presented in the `package.json` file in each created project will be added to the shared bundle using the [Parcel](https://parceljs.org/) bundler. This process is described in great detail here: https://gowebly.org/complete-user-guide/create-new-project#what-cli-does-for-you
